Description: Set the source encoding to fix the build failure with Java 9
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/build.xml
+++ b/build.xml
@@ -31,7 +31,7 @@
   <target name="compile" description="o Compile the code" depends="get-deps">
     <mkdir dir="${classesdir}">
     </mkdir>
-    <javac destdir="${classesdir}" deprecation="true" debug="true" optimize="false" excludes="**/package.html">
+    <javac destdir="${classesdir}" deprecation="true" debug="true" optimize="false" excludes="**/package.html" encoding="ISO-8859-1">
       <src>
         <pathelement location="src/java">
         </pathelement>
@@ -169,7 +169,7 @@
   <target name="compile-tests" depends="compile">
     <mkdir dir="${testclassesdir}">
     </mkdir>
-    <javac destdir="${testclassesdir}" deprecation="true" debug="true" optimize="false" excludes="**/package.html">
+    <javac destdir="${testclassesdir}" deprecation="true" debug="true" optimize="false" excludes="**/package.html" encoding="ISO-8859-1">
       <src>
         <pathelement location="src/test">
         </pathelement>
@@ -195,7 +195,7 @@
     </property>
     <property name="title" value="Avalon LogKit 2.0.dev-0 API">
     </property>
-    <javadoc bottom="Copyright &amp;copy; 2003-2004 The Apache Software Foundation. All Rights Reserved." use="true" private="true" destdir="${javadocdir}" author="true" version="true" sourcepath="src/java" packagenames="org.apache.log.*">
+    <javadoc bottom="Copyright &amp;copy; 2003-2004 The Apache Software Foundation. All Rights Reserved." use="true" private="true" destdir="${javadocdir}" author="true" version="true" sourcepath="src/java" packagenames="org.apache.log.*" encoding="ISO-8859-1">
       <classpath>
         <fileset dir="${libdir}">
           <include name="*.jar">
